includes/view/User_view.php

```diff
@@ -132,7 +173,7 @@ function Users_view(
     $usersList[] = [
         'name'         => '<strong>' . __('Sum') . '</strong>',
         'arrived'      => $arrived_count,
-        'got_voucher'  => $voucher_count,
+        'got_voucher'  => '<div class="text-center">' . $voucher_count . '</div>',
```

Comment:
That could change too on update *duckandhide

Reply:
I can't follow - after which update?

Reply:
I was able to decrypt: After issuing a voucher with an async request, the total number of vouchers at the bottom of the table does still show the old value.

Reply:
Done